test/JDBC/expected/BABEL-CHECK-CONSTRAINT-vu-prepare.out

```diff
@@ -0,0 +1,9 @@
+-- BABEL-3319: Checking that alter table check constraint does not raise collation error
```

Comment:
should we also add upgrade tests to show the behaviour difference before latest version (2.3.0) and after the latest version? 

Also, should we extend existing verify script to show expected behaviour as well? For example,
```
create table foo (a varchar(100) collate sys.database_default);

insert few entries

alter table.. add constraint..

insert few entries which would violate this new constraint 
```

Reply:
\+ creation-time CHECK constraint and case-insensitive insertions.

Reply:
To test different behavior before the latest, you can refer https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/270/files